src/commands/api-mesh/__tests__/log-get-bulk.test.js

```diff
@@ -291,3 +295,249 @@ describe('GetBulkLogCommand startTime and endTime validation', () => {
+			totalSize: 2048,
+		});
+		promptConfirm.mockResolvedValue(true);
+		global.requestId = 'dummy_request_id';
+	});
+
+	afterEach(() => {
+		jest.clearAllMocks();
+	});
+
+	test('runs with valid --past and --from flags', async () => {
+		fs.existsSync.mockReturnValue(true);
+		fs.statSync.mockReturnValue({ size: 0 });
+
+		const mockWriteStream = {
+			write: jest.fn(),
+			end: jest.fn(),
+			on: jest.fn((event, callback) => {
+				if (event === 'finish') {
+					callback();
+				}
+			}),
+		};
+		fs.createWriteStream.mockReturnValue(mockWriteStream);
+
+		const command = new GetBulkLogCommand([], {});
+		await command.run();
+
+		expect(initRequestId).toHaveBeenCalled();
+		expect(initSdk).toHaveBeenCalled();
+		expect(getMeshId).toHaveBeenCalledWith('orgCode', 'projectId', 'workspaceId', 'workspaceName');
+		expect(getPresignedUrls).toHaveBeenCalledWith(
+			'orgCode',
+			'projectId',
+			'workspaceId',
+			'meshId',
+			expect.any(String),
+			expect.any(String),
+		);
+		expect(fs.createWriteStream).toHaveBeenCalledWith(path.resolve(process.cwd(), 'test.csv'), {
+			flags: 'a',
+		});
+		expect(mockWriteStream.write).toHaveBeenCalled();
+		expect(mockWriteStream.end).toHaveBeenCalled();
+	});
+
+	test('throws an error with invalid --from date components', async () => {
+		parseSpy.mockResolvedValueOnce({
+			flags: {
+				past: '20mins',
+				from: '2025-13-01:25:61:61',
+				filename: 'test.csv',
+				ignoreCache: false,
+			},
+		});
+
+		const command = new GetBulkLogCommand([], {});
+		await expect(command.run()).rejects.toThrow(
+			'Found invalid date components passed in --from. Check and correct the date.',
+		);
+	});
+
+	test('throws an error with invalid --from date format', async () => {
+		parseSpy.mockResolvedValueOnce({
+			flags: {
+				past: '15mins',
+				from: '2025:03:01:15:00:00',
+				filename: 'test.csv',
+				ignoreCache: false,
+			},
+		});
+
+		const command = new GetBulkLogCommand([], {});
+		await expect(command.run()).rejects.toThrow(
+			'Invalid format. Use the format YYYY-MM-DD:HH:MM:SS for --from.',
+		);
+	});
+
+	test('runs with valid --past flag without --from', async () => {
+		parseSpy.mockResolvedValueOnce({
+			flags: {
+				past: '15mins',
+				filename: 'test.csv',
+				ignoreCache: false,
+			},
+		});
+
+		fs.existsSync.mockReturnValue(true);
+		fs.statSync.mockReturnValue({ size: 0 });
+
+		const command = new GetBulkLogCommand([], {});
+		await command.run();
+
+		expect(initRequestId).toHaveBeenCalled();
+		expect(initSdk).toHaveBeenCalled();
+		expect(getMeshId).toHaveBeenCalledWith('orgCode', 'projectId', 'workspaceId', 'workspaceName');
+		expect(getPresignedUrls).toHaveBeenCalledWith(
+			'orgCode',
+			'projectId',
+			'workspaceId',
+			'meshId',
+			expect.any(String),
+			expect.any(String),
+		);
+	});
+
+	test('throws an error with edge case for --past duration', async () => {
+		parseSpy.mockResolvedValueOnce({
+			flags: {
+				past: '0s',
+				from: '2025-03-10:12:00:00',
```

Comment:
same comment as above regarding the test failure once we go past 30 days from this date?
Check all places where the from is hard coded and expects to not throw an error